core/frontend/src/IModelApp.ts

```diff
@@ -735,7 +736,7 @@ export class IModelApp {
     return this.makeLogoCard({
       iconSrc: "images/about-imodeljs.svg",
       heading: `<span style="font-weight:normal">${this.i18n.translate("Notices.PoweredBy")}</span>&nbsp;iModel.js`,
-      notice: `${this.applicationVersion}<br>${copyrightNotice}`,
+      notice: `${imodeljsVersion}<br>${copyrightNotice}`,

```

Comment:
Tested this out with both the webpack dev server and a webpack-ed build of display-test-app, and can confirm `require` will work in a production build. See:
![image](https://user-images.githubusercontent.com/11051042/102246000-3d710680-3ecc-11eb-9ef6-4b61f8b7395c.png)

Regarding, creating another variable to keep track of the version number, I agree it's unnecessary, but was what I was recommended to do. I do however think adding this as a property of IModelApp instead might potentially come in handy later with extensions, ie trying to load an extension incompatible with the current IModel.js version of the host app. Granted extensions are internal right now so no need to optimize that just yet.  Let me know what you guys think, and I can update accordingly.

Reply:
Please make the change I suggested to avoid unnecessary work at load time. If we need a way to check version for other reasons, we can make a method for that.

Reply:
@calebmshafer maybe now's the time to change this to say "Powered by iTwin.js"?

Reply:
@kabentley I'd like to hold off and make the switch in the repo all at once to avoid inconsistency.  We should be able to do that in the next couple weeks.